services/app/src/components/views/SplashScreen.tsx

```diff
@@ -48,10 +51,22 @@ class PlayerCounter extends React.Component<PlayerCounterProps, {}> {
       this.setState({transitionTimeout: setTimeout(() => {
         this.props.onPlayerCountSelect(numFingers);
       }, this.props.transitionMillis)});
+      this.animate();
     }
     this.setState({touchCount: numFingers, maxTouches: Math.max(this.state.maxTouches, numFingers)});
   }
 
+  private animate() {
+    if (this.state.transitionTimeout === null) {
+      return;
+    }
+    const progress = (Date.now() - this.state.lastTouchTime) / (this.props.transitionMillis) * 100;
```

Comment:
If performance is an issue, this would be straigthforward to move to being a CSS animation that gets reset by JS via a class change

Reply:
It slows down a little when there's multiple touches, but the performance is roughly the same as what happens the prod webapp.